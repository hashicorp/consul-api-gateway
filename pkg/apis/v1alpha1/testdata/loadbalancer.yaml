apiVersion: api-gateway.consul.hashicorp.com/v1alpha1
kind: GatewayClassConfig
metadata:
  name: test-gateway-class-config
spec:
  serviceType: "LoadBalancer"
---
apiVersion: gateway.networking.k8s.io/v1alpha2
kind: GatewayClass
metadata:
  name: test-gateway-class
spec:
  controller: "hashicorp.com/consul-api-gateway-gateway-controller"
  parametersRef:
    group: api-gateway.consul.hashicorp.com
    kind: GatewayClassConfig
    name: test-gateway-class-config
---
apiVersion: gateway.networking.k8s.io/v1alpha2
kind: Gateway
metadata:
  name: test-loadbalancer
  annotations:
    "external-dns.alpha.kubernetes.io/hostname": "test.example.com"
spec:
  gatewayClassName: test-gateway-class
  listeners:
  - protocol: HTTP
    port: 8080
    name: http
    allowedRoutes:
      namespaces:
        from: Same
  - protocol: HTTPS
    port: 8443
    name: https
    allowedRoutes:
      namespaces:
        from: Same
