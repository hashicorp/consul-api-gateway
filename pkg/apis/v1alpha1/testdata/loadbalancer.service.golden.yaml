metadata:
  creationTimestamp: null
  labels:
    api-gateway.consul.hashicorp.com/managed: "true"
    api-gateway.consul.hashicorp.com/name: test-loadbalancer
    api-gateway.consul.hashicorp.com/namespace: ""
  name: test-loadbalancer
spec:
  ports:
  - name: http
    port: 8080
    protocol: TCP
    targetPort: 0
  - name: https
    port: 8443
    protocol: TCP
    targetPort: 0
  selector:
    api-gateway.consul.hashicorp.com/managed: "true"
    api-gateway.consul.hashicorp.com/name: test-loadbalancer
    api-gateway.consul.hashicorp.com/namespace: ""
  type: LoadBalancer
status:
  loadBalancer: {}
